doc/api-extensions.md

```diff
@@ -2351,3 +2351,8 @@ This field indicates which instance types are supported by the server.
 
 Adds a `mounted` field to disk resources that LXD discovers on the system, reporting whether that disk or partition is
 mounted.
+
+## `server_lts_version_info`
```

Comment:
Lets just use server_version_lts

And you can rebase now too to get the ceph fix.